src/main/java/io/codelex/javaAdvancedTest/Exercise1/CreditCard.java

```diff
@@ -0,0 +1,21 @@
+package io.codelex.javaAdvancedTest.Exercise1;
+
+public class CreditCard extends Card {
+    public CreditCard(int number, String owner, String cvv, double balance) {
+        super(number, owner, cvv, balance);
+    }
+
+    @Override
+    public void addMoney(double money) {
+        if (money > 10000) {
```

Comment:
Šis ne līdz galam atbilst uzdevuma nosacījumiem - tur bija teikts, ka jāpārbauda vai nauda kartē nav > 10000 bet šeit tiek pārbaudīts vai nauda, kuru cenšamies pievienot nav > 10000.